components/HeaderSSR/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { Container } from '@catho/quantum';
+import NavigationMenu from '../NavigationMenu';
+
+const HeaderSSR = () => (
+  <Container>
+    <div className="NavigationBar">
+      <a href="/">
+        <img src="../static/imgs/cathoLogo.png" alt="Logotipo Catho" />
+      </a>
```

Comment:
It could be a component called Logo. If done, place it in a folder called SubComponents within the Header